lib/convection/control/stack.rb

```diff
@@ -85,6 +85,7 @@ def initialize(name, template, options = {})
         @resources = {}
         @current_template = {}
         @last_event_seen = nil
+        @retries = 0
```

Comment:
As far as I remember here, the Stack is only instantiated at startup when the Cloudfile is parsed.